lms/templates/courseware/course_about.html

```diff
@@ -319,6 +319,14 @@
             </li>
           % endif
 
+          % if pre_requisite_course:
+          <li class="prerequisite-course important-dates-item">
+            <i class="icon icon-list-ul"></i>
+            <p class="important-dates-item-title">${_("Requirements")}</p>
+            <span class="important-dates-item-text pre-requisite">${pre_requisite_course}</span>
+            <p class="tip">${_("Access to this course requires the successful completion of ")}${pre_requisite_course}.</p>
```

Comment:
OK. I'd change that to one of these, then:

You must successfully complete `<course>` before you access this course.

You must successfully complete `<course>` before you access materials for this course.

You must successfully complete `<course>` before you begin this course.